css/ampdoc.css

```diff
@@ -367,3 +367,16 @@ amp-autocomplete > textarea,
 [amp-fx^="fly-in"] {
   visibility: hidden;
 }
+
+/**
+ * amp-script[nodom] is an element with no UI, similar to amp-analytics.
+ */
+amp-script[nodom] {
+  /* Fixed to make position independent of page other elements. */
+  position: fixed !important;
+  top: 0 !important;
```

Comment:
why do we need to specify top/overflow?

Reply:
This aligns with CSS for amp-analytics. My guess is `top` helps make sure it's in viewport to be laid out, not sure about `overflow`.

Reply:
I left it out because they didn't seem necessary 🤷